exercises/robot-simulator/src/test/java/RobotTest.java

```diff
@@ -203,7 +207,7 @@ public void testAdvancingWhenFacingWestDecreasesXCoordinateByOne() {
     @Test
     public void testInstructionsToMoveWestAndNorth() {
         final Robot robot = new Robot(new GridPosition(0, 0), Orientation.NORTH);
-
+        
```

Comment:
Empty space on this line can be removed 🙂

Reply:
Is it just line 210? Or would you also like to get rid of the whitespace on line 224 & 238 @sjwarner-bp?

Reply:
224 etc is fine I think, as they don't have extra space or tab characters in the lines.

The line breaks are absolutely fine, and I think they enhance readability - it's just if there is some trailing whitespace :slightly_smiling_face:

Reply:
Ah. I see what you mean now. Thanks!